scipy/signal/tests/test_filter_design.py

```diff
@@ -484,6 +491,12 @@ def test_vs_freqs(self):
         assert_allclose(w1, w2)
         assert_allclose(h1, h2, rtol=1e-6)
 
+    def test_backward_compat(self):
+        # For backward compatibility, test if None act as a wrapper for default
+        w1, h1 = freqs_zpk([1.0],[1.0],[1.0])
+        w2, h2 = freqs_zpk([1.0],[1.0],[1.0],None)
```

Comment:
Just a small nitpick: To improve readability commas are usually followed by a whitespace, e.g. 
`w2, h2 = freqs_zpk([1.0], [1.0], [1.0], None)`. This style is used throughout SciPy and in general recommended by most of Pythons style guides. :wink: